btrfs test fail btrfs-prog 4.17

I'm seeing errors when running the containerd test on a alpine 3.8 system which uses btrfs-progs v4.17:
```
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547605695Z: ok  	github.com/containerd/containerd/archive	2.522s
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547613468Z: ok  	github.com/containerd/containerd/metadata	7.950s
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547621464Z: ok  	github.com/containerd/containerd/mount/lookup_test	1.528s
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547629023Z: --- FAIL: TestBtrfs (0.01s)
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547640633Z:     --- FAIL: TestBtrfs/Basic (0.43s)
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547650546Z:     	testsuite.go:92: Failed to initialize snapshotter: exit status 1
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547659960Z:     		failed to make btrfs filesystem (out: "btrfs-progs v4.17 \nERROR: '/dev/loop1' is too small to make a usable filesystem\nERROR: minimum size for each btrfs device is 114294784\nSee http://btrfs.wiki.kernel.org for more information.\n\n")
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547668845Z:     		github.com/containerd/containerd/snapshots/btrfs.boltSnapshotter.func1
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547676403Z:     			/go/src/github.com/containerd/containerd/snapshots/btrfs/btrfs_test.go:62
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547684129Z:     		github.com/containerd/containerd/snapshots/testsuite.makeTest.func1
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547695384Z:     			/go/src/github.com/containerd/containerd/snapshots/testsuite/testsuite.go:90
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547705863Z:     		testing.tRunner
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547713741Z:     			/usr/lib/go/src/testing/testing.go:777
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547723779Z:     		runtime.goexit
[+ 54m 30s] [STDERR  ] 2018-07-07T19:28:03.547731590Z:     			/usr/lib/go/src/runtime/asm_amd64.s:2361
```

I see the same error for all `btrfs` tests. Basically the [`containerd test code`](https://github.com/containerd/containerd/blob/master/snapshots/btrfs/btrfs_test.go#L48) is creating a 100MB loop back device but the btrfs-progs v4.17 tools seem to expect at least a 114294784B (109MB) device.

I suspect [this commit](https://git.kernel.org/pub/scm/linux/kernel/git/kdave/btrfs-progs.git/commit/?id=997f9977c24397eb6980bb91e7f9cbc6fe21815a) introduced with v4.15 is causing this.

AN obvious quick fix is to increase the size of the loop back [here](https://github.com/containerd/containerd/blob/master/snapshots/btrfs/btrfs_test.go#L48). Happy to provide a patch.

--- a/snapshots/btrfs/btrfs_test.go
+++ b/snapshots/btrfs/btrfs_test.go
@@ -45,7 +45,7 @@ func boltSnapshotter(t *testing.T) func(context.Context, string) (snapshots.Snap
 
 	return func(ctx context.Context, root string) (snapshots.Snapshotter, func() error, error) {
 
-		loopbackSize := int64(100 << 20) // 100 MB
+		loopbackSize := int64(128 << 20) // 128 MB
 		// mkfs.btrfs creates a fs which has a blocksize equal to the system default pagesize. If that pagesize
 		// is > 4KB, mounting the fs will fail unless we increase the size of the file used by mkfs.btrfs
 		if os.Getpagesize() > 4096 {

